metagpt/provider/bedrock/utils.py

```diff
@@ -1,52 +1,84 @@
-    "ai21.j2-ultra-v1": 8000,
+NOT_SUPPORT_STREAM_MODELS = {
+    # Jurassic-2 Mid-v1 and Ultra-v1
+    # + Legacy date: 2024-04-30 (us-west-2/Oregon)
+    # + EOL date: 2024-08-31 (us-west-2/Oregon)
+    "ai21.j2-mid-v1": 8191,
+    "ai21.j2-ultra-v1": 8191,
 }
 
 SUPPORT_STREAM_MODELS = {
+    # Jamba-Instruct
+    "ai21.jamba-instruct-v1:0": 4096,
+    # Titan Text G1 - Express
+    "amazon.titan-text-express-v1": 8192,
+    "amazon.titan-text-express-v1:0:8k": 8192,
+    # Titan Text G1 - Lite
+    "amazon.titan-text-lite-v1:0:4k": 4096,
+    "amazon.titan-text-lite-v1": 4096,
+    # Titan Text Premier
+    "amazon.titan-text-premier-v1:0": 3072,
+    "amazon.titan-text-premier-v1:0:32k": 3072,
+    # Titan Text Large
     "amazon.titan-tg1-large": 8000,
-    "amazon.titan-text-express-v1": 8000,
-    "amazon.titan-text-express-v1:0:8k": 8000,
-    "amazon.titan-text-lite-v1:0:4k": 4000,
-    "amazon.titan-text-lite-v1": 4000,
-    "anthropic.claude-instant-v1": 100000,
-    "anthropic.claude-instant-v1:2:100k": 100000,
-    "anthropic.claude-v1": 100000,
-    "anthropic.claude-v2": 100000,
-    "anthropic.claude-v2:1": 200000,
-    "anthropic.claude-v2:0:18k": 18000,
-    "anthropic.claude-v2:1:200k": 200000,
-    "anthropic.claude-3-sonnet-20240229-v1:0": 200000,
-    "anthropic.claude-3-sonnet-20240229-v1:0:28k": 28000,
-    "anthropic.claude-3-sonnet-20240229-v1:0:200k": 200000,
-    "anthropic.claude-3-haiku-20240307-v1:0": 200000,
-    "anthropic.claude-3-5-sonnet-20240620-v1:0": 200000,
-    "anthropic.claude-3-haiku-20240307-v1:0:48k": 48000,
-    "anthropic.claude-3-haiku-20240307-v1:0:200k": 200000,
-    # currently (2024-4-29) only available at US West (Oregon) AWS Region.
-    "anthropic.claude-3-opus-20240229-v1:0": 200000,
-    "cohere.command-text-v14": 4000,
-    "cohere.command-text-v14:7:4k": 4000,
-    "cohere.command-light-text-v14": 4000,
-    "cohere.command-light-text-v14:7:4k": 4000,
-    "meta.llama2-13b-chat-v1:0:4k": 4000,
-    "meta.llama2-13b-chat-v1": 2000,
-    "meta.llama2-70b-v1": 4000,
-    "meta.llama2-70b-v1:0:4k": 4000,
-    "meta.llama2-70b-chat-v1": 2000,
-    "meta.llama2-70b-chat-v1:0:4k": 2000,
-    "meta.llama3-8b-instruct-v1:0": 2000,
-    "meta.llama3-70b-instruct-v1:0": 2000,
-    "mistral.mistral-7b-instruct-v0:2": 32000,
-    "mistral.mixtral-8x7b-instruct-v0:1": 32000,
-    "mistral.mistral-large-2402-v1:0": 32000,
+    # Claude Instant v1
+    "anthropic.claude-instant-v1": 4096,
+    "anthropic.claude-instant-v1:2:100k": 4096,
+    # Claude v2
+    "anthropic.claude-v2": 4096,
+    "anthropic.claude-v2:0:18k": 4096,
+    "anthropic.claude-v2:0:100k": 4096,
+    # Claude v2.1
+    "anthropic.claude-v2:1": 4096,
+    "anthropic.claude-v2:1:18k": 4096,
+    "anthropic.claude-v2:1:200k": 4096,
+    # Claude 3 Sonnet
+    "anthropic.claude-3-sonnet-20240229-v1:0": 4096,
+    "anthropic.claude-3-sonnet-20240229-v1:0:28k": 4096,
+    "anthropic.claude-3-sonnet-20240229-v1:0:200k": 4096,
```

Comment:
is it(4096) rights? maybe you should have a check with other models.

Reply:
`max_tokens`_to_sample is 4096 for every model, except for Claude 3.5 Sonnet (8192)

**References:**
* (AWS) [Anthropic Claude Text Completions API > Request and Response](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-anthropic-claude-text-completion.html#model-parameters-anthropic-claude-text-completion-request-response)
* (Anthropic) [Model comparison](https://docs.anthropic.com/en/docs/about-claude/models#model-comparison)